src/patternfly/base/tokens/_tokens-charts.scss

```diff
@@ -1,14 +1,14 @@
 // /**
 //  * Do not edit directly
-//  * Generated on Tue, 20 Feb 2024 01:08:05 GMT
+//  * Generated on Thu, 21 Mar 2024 17:46:16 GMT
 //  */
 
 :root {
-  --pf-t--chart--stroke--width--sm: 2px;
-  --pf-t--chart--stroke--width--xs: 1px;
-  --pf-t--chart--BorderWidth--lg: 8px;
-  --pf-t--chart--BorderWidth--sm: 2px;
-  --pf-t--chart--BorderWidth--xs: 1px;
+  --pf-t--chart--stroke--width--sm: 2;
+  --pf-t--chart--stroke--width--xs: 1;
+  --pf-t--chart--BorderWidth--lg: 8;
+  --pf-t--chart--BorderWidth--sm: 2;
+  --pf-t--chart--BorderWidth--xs: 1;
```

Comment:
@lboehling just want to confirm this is an intended change? Pretty sure it's a good change from our side, since these tokens are

Reply:
It's exported as 
```    "BorderWidth": {
      "xs": {
        "type": "number",
        "value": 1
      },
      "sm": {
        "type": "number",
        "value": 2
      },
      "lg": {
        "type": "number",
        "value": 8
      }
    },
    "stroke": {
      "width": {
        "xs": {
          "type": "number",
          "value": 1
        },
        "sm": {
          "type": "number",
          "value": 2
        }
      }
    },
``` 
and I thought you wanted unitless, so the dimension isn't added by the tranformer right now.

Reply:
@srambach perfect! Just wanted to verify it was intentional 👍